### PR TITLE
feat: Create Geolocation package [Geolocation] (#55)

### PR DESCRIPTION
**Pull request by Edilberto-Vazquez**
## Summary
Geolocation package was created for the squad geolocation service. This solve issue #55

### CODE REVIEW

**Review by CETHER — APPROVED** [review]
The changes are ok.

**Review by emanuelalvaradog — APPROVED** [review]
Changes are ok

**Review by emanuelalvaradog — COMMENTED** [review]
Everything looking fine

**Comment by DilanAriza** [comment]
Everything looking fine

**Review by DilanAriza — APPROVED** [review]
Everything looking fine

**Review by DilanAriza — COMMENTED** [review]
Everything looking fine

**Review by DilanAriza — APPROVED** [review]
Everything looking fine

**Review by emanuelalvaradog — DISMISSED** [review]
Done

**Review by DilanAriza — APPROVED** [review]

**Review by emanuelalvaradog — COMMENTED** [review]
Done

**Review by DilanAriza — APPROVED** [review]

**Review by DilanAriza — APPROVED** [review]

**Review by DilanAriza — APPROVED** [review]

**Review by CETHER — DISMISSED** [review]

**Review by DilanAriza — APPROVED** [review]

**Review by DilanAriza — APPROVED** [review]

File: packages/geolocation/src/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+const welcome = require("./utils/welcomePackage")
+
+module.exports = {
+    geolocationWelcome: welcome,
+}
```
Comment (alexcamachogz) [inline packages/geolocation/src/index.js:7]: The linter had left you this message:  “No newline at end of file”, make sure to follow this kind of recommendation.

File: packages/geolocation/src/utils/welcomePackage.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const geolocationWelcome = () => {
+    console.log("Welcome to geolocation squad 😂🧭")
+}
+
+module.exports = geolocationWelcome
```
Comment (alexcamachogz) [inline packages/geolocation/src/utils/welcomePackage.js:5]: Same case here

**Review by alexcamachogz — CHANGES_REQUESTED** [review]
Apply all linter recommendation to your files, make sure all your files follow this kind of rules.

File: packages/geolocation/src/index.js (hunk 1 above)
Comment (Edilberto-Vazquez) [reply packages/geolocation/src/index.js]: ready I already add the line break

File: packages/geolocation/src/utils/welcomePackage.js (hunk 2 above)
Comment (Edilberto-Vazquez) [reply packages/geolocation/src/utils/welcomePackage.js]: ready I already add the line break

**Review by alexcamachogz — DISMISSED** [review]
Everything looks good to me 👌🏻

**Review by CETHER — APPROVED** [review]
Everything looks ok!

**Pull request merged**